README.md

```diff
@@ -3,29 +3,79 @@
 
 This plugin allows you to test **Electron** applications with [TestCafe](http://devexpress.github.io/testcafe).
 
-## Install
+## Getting Started
+
+### Installation
 
 ```
 npm install testcafe-browser-provider-electron
 ```
 
-## Usage
+### Testing a JavaScript Application
 
-First, create a `.testcafe-electron-rc` file in the root directory of your Electron app. For more info, see the [Configuration](#configuration) section.
+If you have a JavaScript application that runs in Electron, you need to go through the following steps to set up testing.
 
-```
-{
-  "mainWindowUrl": "./index.html"
-}
-```
+1. Create a `.testcafe-electron-rc` file in the root application directory. Include the following settings to this file.
 
-When you run tests from the command line, specify the path to the application root directory prefixed with "electron:" :
```

Comment:
In the command line specify the path to the app root directory. Prefix it with "electron:":